jobs/integr8ly/ocp4/nightly/osd-ocp4-testing-nightly-trigger.yaml

```diff
@@ -22,15 +22,18 @@
             REGISTRY_NAMESPACE=integreatly
             INSTALLATION_TYPE=managed
             ADMIN_USERNAME=kubeadmin
+            CUSTOMER_ADMIN_USERNAME=customer-admin
+            CUSTOMER_ADMIN_PASSWORD=Password1
             GH_CLIENT_ID=yourclientsecret
             GH_CLIENT_SECRET=yourclientsecret
             TEST_SUITES_REPOSITORY=https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
-            TEST_SUITES_BRANCH=master
+            TEST_SUITES_BRANCH=v2-master
             RECIPIENTS=integreatly-qe@redhat.com
             COMPUTE_REPLICAS=3
             CONTROL_PLANE_REPLICAS=3
             AWS_REGION=eu-west-1
             CREDENTIALS_ID=iam-openshift4-credentials
+            NAMESPACE_PREFIX=openshift-
             TO_DO=provision + install + deprovision
```

Comment:
If I got the intended flow right the `provision + install + tests + deprovision` should be here.

Reply:
good point. WIll change it